### ISSUE TITLE
Consider removal of uiowa theme within uiowa.edu site directory

### ISSUE THREAD

**Issue by joewhitsitt** [uiowa/uiowa#7047]
# Reference
https://github.com/uiowa/uiowa/tree/main/docroot/sites/uiowa.edu/themes/uiowa

https://github.com/uiowa/uiowa/pull/6994#issuecomment-1779564356

There are styles still being called from the uiowa_edu_core module and when trying to move it out of the theme, the CSS weight seems to be different and not take precedence. That needs to be accounted for. Otherwise, it sounds like this theme is not used. It is definitely not enabled.

**Comment by joewhitsitt** [comment]
Looking over https://github.com/uiowa/uiowa/pull/9744, it appears we don't want to do this issue. This are the site specific styles, just organized differently. Correct @bspeare

**Comment by bspeare** [comment]
Correct, we still need this custom theme.  It might be worth testing if we still need the template files, but we probably still need them for not displaying the site name.

**Comment by joewhitsitt** [comment]
I feel this issue was created based on misunderstanding so I will close for now. Thanks for chiming in.
